@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -183,6 +183,11 @@ export interface BaseTechniqueParams {
      */
     renderOrder: number;
 
+    /**
+     * The category of this technique.
```

Comment:
Add comment that it references `Theme.priorities`, stg like this:

> Sets  `prority` (`renderOrder`) based on index of this category in [[Theme.priorities]].

Reply:
i added some documentation explaining how to use priorities and labelPriorities. We will drop renderOrder so i'm not sure we should reference it in the documentation for `priorities`